ui/app/src/components/LoggingLevelsManagement/LoggingLevelsManagement.tsx

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (C) 2007-2021 Crafter Software Corporation. All Rights Reserved.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as published by
+ * the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { useCallback, useEffect, useMemo, useState } from 'react';
+import { Logger } from '../../models/Logger';
+import ApiResponse from '../../models/ApiResponse';
+import { fetchLoggers as fetchLoggersService, setLogger } from '../../services/logs';
+import GlobalAppToolbar from '../GlobalAppToolbar';
+import { FormattedMessage } from 'react-intl';
+import { useLogicResource } from '../../utils/hooks';
+import { SuspenseWithEmptyState } from '../SystemStatus/Suspencified';
+import LoggingLevelsGridUI, { LoggingLevelsGridSkeletonTable } from '../LoggingLevelsGrid';
+
+export type LoggerLevel = 'debug' | 'warn' | 'info' | 'error';
```

Comment:
This type belongs in its own file in models.
Both here and the service should employ the use of this type.